Objects/Converters/ConverterCSI/ConverterCSIShared/PartialClasses/Results/ConvertResults.cs

```diff
@@ -1,86 +1,119 @@
-﻿using CSiAPIv1;
-using Objects.Structural.Geometry;
-using Objects.Structural.Results;
-using Speckle.Core.Models;
+using ConverterCSIShared;
+using ConverterCSIShared.Models;
+using Objects.Structural.Loading;
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
 
 namespace Objects.Converter.CSI
 {
   public partial class ConverterCSI
   {
-    public Base ResultsToSpeckle()
+    public void ResultsToSpeckle()
```

Comment:
Same comment as above. If this is a top-level conversion, this should actually return something or else the main `ConvertToSpeckle` will succeed and return null at the same time.

I do feel like I'm missing a key part of this to understand this change.